File: Selenium/Activities/Activity_12.java
package test_basic_practice;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity_12 {

	public static void main(String[] args) {
		WebDriver drive =new ChromeDriver();
		drive.get("https://v1.training-support.net/selenium/dynamic-controls");
		System.out.println("Title page is : "+drive.getTitle());
		WebElement Textinput=drive.findElement(By.id("input-text"));
		System.out.println("Text input status "+Textinput.isEnabled());
		if (Textinput.isEnabled()==false) {
			drive.findElement(By.id("toggleInput")).click();
			System.out.println("Text input status "+Textinput.isEnabled());
		}
		drive.quit();
    }

	}

File: Selenium/Activities/Activity_12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions
from selenium.webdriver.common.action_chains import ActionChains
from selenium.webdriver.common.keys import Keys

with webdriver.Firefox() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("Title is: ",driver.title)

    Textinput=driver.find_element(By.ID,"input-text")
    print("Text input status ",Textinput.is_enabled())
    if Textinput.is_enabled()==False:
        driver.find_element(By.ID,"toggleInput").click()
    print("Text input status ",Textinput.is_enabled())